theme/js/components/discussions/thread.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="thread-wrapper" :id="discussionUrl(id)">
+    <header class="thread-header">
+      <div class="thread-status">
+        <span v-if="closed">Discussion fermée</span>
+      </div>
+      <div class="thread-title">{{ title }}</div>
+      <div class="thread-link">
+        <a :href="discussionUrl(id, true)" v-html="LinkIcon"></a>
+      </div>
+    </header>
+    <div class="thread-content">
+      <transition-group name="list">
+        <article
+          v-for="(comment, index) in _discussion"
+          v-if="!_collapsed"
+          class="thread-comment"
+          :id="commentUrl(id, index)"
+          :key="commentUrl(id, index)"
+        >
+          <avatar :user="comment.posted_by"></avatar>
+          <div class="thread-box">
+            <strong class="author">
+              {{
+                comment.posted_by.first_name + " " + comment.posted_by.last_name
+              }}
+            </strong>
+            <p class="m-0">{{ comment.content }}</p>
+          </div>
+          <div class="thread-link">
+            <span class="thread-date">{{ formatDate(comment.posted_on) }}</span>
+            <a :href="commentUrl(id, index, true)" v-html="LinkIcon"></a>
+          </div>
+        </article>
+      </transition-group>
+      <article
+        class="thread-collapse"
+        v-if="_collapsed"
+        @click.prevent="collapsed = false"
+      >
+        {{ _discussion.length }} messages
+      </article>
+    </div>
+    <footer class="thread-footer">
+      <div v-if="!closed">
+        <div
+          class="thread-reply-cta"
+          v-if="!showForm"
+          @click.stop="displayForm"
+        >
+          + Ajouter un commentaire
+        </div>
+        <thread-reply
+          :subjectId="id"
+          v-if="showForm"
+          :onSubmit="replyToThread"
+        />
+      </div>
+      <div v-if="closed" class="text-grey-300">
+        La discussion a été close par&#32;
+        <span class="text-blue-200 px-xxs">{{
+          closed_by.first_name + " " + closed_by.last_name
+        }}</span>
+        le {{ formatDate(closed) }}
+      </div>
+    </footer>
+  </div>
+</template>
+
+<script>
+import ThreadReply from "./thread-reply.vue";
+import Avatar from "./avatar.vue";
+import LinkIcon from "svg/permalink.svg";
+import dayjs from "dayjs";
+import "dayjs/locale/fr";
+
+dayjs.locale("fr");
```

Comment:
This should be configurable. We currently have a JS variable called `uData.config.lang` that might do the trick.